server/test/unit/com/thoughtworks/go/server/controller/AgentRegistrationControllerTest.java

```diff
@@ -65,7 +65,6 @@ public void setUp() throws Exception {
         goConfigService = mock(GoConfigService.class);
 
         when(systemEnvironment.getSslServerPort()).thenReturn(8443);
-        when(systemEnvironment.get(SystemEnvironment.TFS_SDK_10)).thenReturn(true);
```

Comment:
Won't removing that expectation break any of the tests? May be trigger a PR build, but also run the tests locally for quick feedback